services/ui_backend_service/cache/store.py

```diff
@@ -101,7 +101,7 @@ async def get_recent_run_numbers(self):
             values=[int(round(time.time() * 1000)) - (int(METAFLOW_ARTIFACT_PREFETCH_RUNS_SINCE) * 1000)],
             order=['ts_epoch DESC'],
             limit=METAFLOW_ARTIFACT_PREFETCH_RUNS_LIMIT,
-            expanded=True
+            expanded=FEATURE_MODEL_EXPAND
```

Comment:
wasn't this used only internally for preloading data for runs? in this case it could be left to always expanded, so it uses run numbers internally.

Reply:
True, will change it back